app/views/admin/suppliers/_layout.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1 class="govuk-heading-l">Suppliers</h1>
+<div class="page-tab-container">
+  <div class="page-tab govuk-!-margin-right-8 <%= "page-tab__selected" if page == "suppliers" %>">
+    <% if page == "suppliers" %>
+      All suppliers
+    <% else %>
+      <%= govuk_link_to "All suppliers", admin_suppliers_path %>
+    <% end %>
+  </div>
```

Comment:
This is like a page navbar, right? Kind of like the one on https://design-system.service.gov.uk/components/?

If it makes any difference, they don't bother with making current element not a link - and it looks a bit odd to see two if statements checking the same thing in two consecutive lines.

Reply:
I take issue with that navigation: there is no indication of the current page to a screen reader user. @cfq do you know of good patterns for this?

Reply:
I expect something like leaving it as a link and adding a `visually-hidden` span saying "Current page" or similar might work?

Reply:
I think it would, yes :)

Reply:
How's this @kicferk1 ?